dScripts/EquipmentScripts/PersonalFortress.cpp

```diff
@@ -2,49 +2,56 @@
 #include "GameMessages.h"
 #include "SkillComponent.h"
 #include "DestroyableComponent.h"
+#include "ControllablePhysicsComponent.h"
 #include "EntityManager.h"
 
 void PersonalFortress::OnStartup(Entity* self) {
 	auto* owner = self->GetOwner();
 	self->AddTimer("FireSkill", 1.5);
-	GameMessages::SendSetStunned(owner->GetObjectID(), PUSH, owner->GetSystemAddress(), LWOOBJID_EMPTY,
-		true, true, true, true, true, true, true, true, true
-	);
 
-	auto* destroyableComponent = owner->GetComponent<DestroyableComponent>();
+	auto* destroyableComp = owner->GetComponent<DestroyableComponent>();
+	if (destroyableComp) destroyableComp->SetStatusImmunity(
+			eStateChangeType::PUSH,
+			true, true, true, true, true, false, true, false, false
```

Comment:
Im seeing a mismatch in immunity pushes here.  Can you elaborate on the change?

Reply:
this is in line with the script from live, since it was a small script and made use of the immunities, and since I broke it, it was good to test with.
I just double checked that this lines up with the lua script, and it does